ote/src/clj/ote/services/transport_operator.clj

```diff
@@ -317,30 +317,41 @@
         (log/warn (str "Error while inviting " user-email " ") e)))))
 
 (defn manage-adding-users-to-operator [email db requester operator form-data]
-  (let [authority? (= (::t-operator/group-id operator) (transit-authority-group-id db))
-        new-member (first (fetch-user-by-email db {:email (:email form-data)}))
-        ckan-group-id (::t-operator/group-id operator)
-        operator-users (fetch-operator-users db {:ckan-group-id ckan-group-id})
-        not-invited? (empty?
-                       (specql/fetch db ::user/user-token
-                                     #{::user/user-email}
-                                     {::user/ckan-group-id ckan-group-id
-                                      ::user/user-email (:email form-data)
-                                      ::user/expiration (op/>= (tc/to-sql-date (t/now)))}))
+  (let [transit-authority?            (= (::t-operator/group-id operator) (transit-authority-group-id db))
+        allowed-to-manage?            (and (authorization/admin? requester)
```

Comment:
@rckopteri So one thing I'm not that sure about, when user is referred to as admin, in database the field is called "sysadmin". Should I treat it as overriding property wherein sysadmins can do everything or is this "_admin + something else_" more apt way to check this?